sympy/functions/combinatorial/numbers.py

```diff
@@ -841,6 +841,116 @@ def _eval_evalf(self, prec):
         if self.args[0].is_number:
             return self.rewrite(C.gamma)._eval_evalf(prec)
 
+
+#----------------------------------------------------------------------------#
+#                                                                            #
+#                           Genocchi numbers                                 #
+#                                                                            #
+#----------------------------------------------------------------------------#
+
+
+class genocchi(Function):
+    r"""
```

Comment:
No 'r' before a string means, treating it as raw string. All escape sequences will be ignored